### PR TITLE
Fix/Add winit&egl logging

### PR DESCRIPTION
**Pull request by Drakulix**
- [x] Added logging to winit + egl
- [x] Fixed a small some small copy&paste message errors in the libinput backend
- [x] Fixed winit backend only sending horizontal pointer axis events
- [x] Added support for double buffering an egl context (I should do a glutin pr, if this works)

### CODE REVIEW

File: src/backend/winit.rs [hunk 1]
```diff
@@ -156,6 +178,7 @@ impl EGLGraphicsBackend for WinitGraphicsBackend {
     }
 
     unsafe fn make_current(&self) -> Result<(), SwapBuffersError> {
+        debug!(self.logger, "Making context the current one");
```
Comment (vberger) [inline src/backend/winit.rs]: I can't parse this sentence, maybe a word is missing?

**Comment by vberger** [comment]
LGTM modulo the small nit. 👍

File: src/backend/winit.rs (hunk 1 above)
Comment (Drakulix) [inline src/backend/winit.rs]: Yes that is kinda bad. Changed it.

**Comment by vberger** [comment]
👍

**Pull request merged**